config/environments/production.rb

```diff
@@ -73,8 +73,8 @@
 
   # Enable locale fallbacks for I18n (makes lookups for any locale fall back to
   # the I18n.default_locale when a translation can not be found)
-  config.i18n.fallbacks = true
-
+  #config.i18n.fallbacks = true
+  config.i18n.fallbacks = {:en => [:en, :de], :de => [:de, :en]}
```

Comment:
Cool, did this work for you in your local copy?

Reply:
I tried setting up the system in production mode on my system and failed terribly. There were multiple failing migrations. I tried skipping them but because of these skipped migrations many later migrations also failed. Could we test it in staging?